scripts/appleseedMaya/renderGlobals.py

```diff
@@ -405,20 +406,24 @@ def create(self):
                         adaptiveSampling = mc.getAttr("appleseedRenderGlobals.adaptiveSampling")
 
                         self._addControl(
-                            ui=pm.intFieldGrp(
-                                label="Min Samples", numberOfFields=1, enable=adaptiveSampling),
+                            ui=pm.intSliderGrp(
+                                label="Min Samples", field=True, value=16, cw=(3,160), minValue=0, 
+                                fieldMinValue=0, maxValue=256, fieldMaxValue=1000000, enable=adaptiveSampling),
                             attrName="minPixelSamples")
                         self._addControl(
-                            ui=pm.intFieldGrp(
-                                label="Max Samples", numberOfFields=1),
+                            ui=pm.intSliderGrp(
+                                label="Max Samples", field=True, value=256, cw=(3,160), minValue=16, 
+                                fieldMinValue=0, maxValue=1024, fieldMaxValue=1000000),
                             attrName="samples")
                         self._addControl(
-                            ui=pm.intFieldGrp(
-                                label="Batch Sample Size", numberOfFields=1, enable=adaptiveSampling),
+                            ui=pm.intSliderGrp(
+                                label="Batch Sample Size", field=True, value=8, cw=(3,160), minValue=1, 
```

Comment:
Default should be 16.

Reply:
True, this is overridden in the c++ file to 16. I will update it here as well.